src/Loader.php

```diff
@@ -61,6 +69,11 @@ private function loadXml(SimpleXMLElement $root)
         return $graph;
     }
 
+    /**
+     * @param \SimpleXMLElement $xml
+     * @param \Fhaculty\Graph\Attribute\AttributeAware $target
+     * @param array $keys
+     */
```

Comment:
Is much gained by adding these to private methods that use the exact same type hints? (would love to hear some more thoughts on this)